pkg/importer/format-readers.go

```diff
@@ -31,16 +31,15 @@ import (
 
 	"kubevirt.io/containerized-data-importer/pkg/common"
 	"kubevirt.io/containerized-data-importer/pkg/image"
-	"kubevirt.io/containerized-data-importer/pkg/monitoring"
 	"kubevirt.io/containerized-data-importer/pkg/util"
 	prometheusutil "kubevirt.io/containerized-data-importer/pkg/util/prometheus"
 )
 
 var (
 	progress = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
-			Name: monitoring.InternalMetricOptsList[monitoring.CloneProgress].Name,
-			Help: monitoring.InternalMetricOptsList[monitoring.CloneProgress].Help,
+			Name: "clone_progress",
```

Comment:
:+1:

Reply:
This metric is internal, so at the moment we decided to leave it as is. 
In the next refactor steps it will be handled.

Reply:
ofc, I just commented with a thumbs up about it not being included as first class metric